modules/secrets-store-vault.adoc

```diff
@@ -0,0 +1,384 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="secrets-store-vault_{context}"]
+= Mounting secrets from HashiCorp Vault
+
+You can use the {secrets-store-operator} to mount secrets from HashiCorp Vault to a CSI volume in {product-title}.
+
+// TODO: Is there a requirement for particular provider? https://issues.redhat.com/browse/OSDOCS-10136 steps say Azure, is this true?
+// To mount secrets from HashiCorp Vault, your cluster must be installed on Microsoft Azure.
```

Comment:
Update once we hear how the testing went on AWS. If AWS too, will say must be installed on either AWS or Azure.

Reply:
Yesh with vault-provider it works on both AWS/Azure.

Reply:
Great thanks, will update to mention this